modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -257,6 +266,71 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-169960. Verify that a submited MDF will be displayed on the list"
+	@priority = 5
+	test CanViewMDFStatusOnListing {
+		task ("Given I go to add a new MDF Request and when I fill all the goals") {
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Test Description",
+				campaignName = "First Campaign",
+				countryName = "Global",
+				goalsOptions = "Thought leadership",
+				targetMarketOptions = "Aerospace & Defense,Energy",
+				targetRolesOptions = "Director/Manager");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And When I add a new activity") {
+			PRMMDFRequest.newActivity(
+				activityDescription = "Activity Description",
+				activityName = "Activity Name 1",
+				budgetList = "Sponsorship Fee-2000,Room Rental-3000",
+				followUpStrategy = "test",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetOfLeads = "test",
+				typeOfActivity = "Miscellaneous Marketing");
+
+			Button.click(button = "Continue");
+
+			Button.click(button = "Continue");
+
+			Button.click(button = "Submit");
```

Comment:
Hi @mateussandes23, here you should use the macro `PRMMDFRequest.createNewRequest`, like [this example](https://github.com/pt-liferay-solutions/liferay-portal/pull/3254/commits/148e4897327e4581b7d8582342a5d103e04cc204#diff-e11945b2c04a8939c4989e0fb72ed18189ee0dc2db85f67aadbd93fbd74a6f8dR294-R310).